src/DatabaseMetadata/Table.php

```diff
@@ -4,124 +4,78 @@
 
 namespace Keboola\DbExtractorCommon\DatabaseMetadata;
 
-class Table implements \JsonSerializable
+class Table implements \JsonSerializable, ToArrayInterface
 {
     /** @var string */
     private $name;
 
-    /** @var string|null */
+    /** @var string */
     private $sanitizedName;
 
-    /** @var string|null */
+    /** @var string */
     private $schema;
 
-    /** @var string|null */
+    /** @var string */
     private $type;
 
-    /** @var int|null */
```

Comment:
tyhle jsou vyhozené protože je to mysql specific?

Reply:
Presne tak